src/sxapi/base.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+# coding: utf8
+
+import requests
+
+PUBLIC_API_V2 = "https://api.smaxtec.com/api/v2"
+INTEGRATION_API_V2 = "https://api.smaxtec.com/integration/v2"
+
+
+class BaseAPI(object):
+    def __init__(self, base_url, email=None, password=None, api_key=None):
+        """Initialize a new base low level API client instance.
+        """
+        self.api_base_url = base_url.rstrip("/")
+        self.email = email
+        self.password = password
+        self.api_key = api_key
```

Comment:
is this meant to be a token? eg a bearer token?
can we use consistent naming of this?

why do we need api_key, _session_key and api_token?
can we just name it api_token?

Reply:
yes we can ;)

Reply:
I think it was the only leftover from the old code